From 555d823492b7e772249f4f303be978152e96d573 Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Sun, 5 Jan 2020 18:42:51 -0500
Subject: [PATCH 4/5] improve-librtlsdr-pc-file

librtlsdr.pc should declare -lusb-1.0 in Libs.private section
to exclude usb library from dynamic linking.
References to libusb headers are not needed in Cflags, since these headers
are not used by external rtlsdr API, but this is optional.
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=784912
---
 librtlsdr.pc.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/librtlsdr.pc.in b/librtlsdr.pc.in
index 5e55049..52a37e6 100644
--- a/librtlsdr.pc.in
+++ b/librtlsdr.pc.in
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: RTL-SDR Library
 Description: C Utility Library
 Version: @VERSION@
-Cflags: -I${includedir}/ @RTLSDR_PC_CFLAGS@
-Libs: -L${libdir} -lrtlsdr -lusb-1.0
-Libs.private: @RTLSDR_PC_LIBS@
+Cflags: -I${includedir}
+Libs: -L${libdir} -lrtlsdr
+Libs.private:  -lusb-1.0 @RTLSDR_PC_LIBS@
-- 
2.20.1

